src/utils/versions/getVersionsColors.ts

```diff
@@ -12,6 +12,7 @@ export const hashCode = (s: string) => {
 // TODO: colors used in charts as well, need to move to constants
 // 11 distinct colors from https://mokole.com/palette.html
 export const COLORS = [
+    '#008000', // green
```

Comment:
Before all the changes, there were 11 colors (see comment in the code above). Let's not change the number of colors.